tests/e2e/utils/src/components.js

```diff
@@ -352,6 +352,31 @@ const createSimpleOrder = async ( orderStatus = 'Pending payment' ) => {
 	return variablePostIdValue;
 };
 
+/**
+ * Creates a batch of orders from the given `statuses`
+ * using the "Batch Create Order" API.
+ * 
+ * @param statuses Array of order statuses
+ */
+const batchCreateOrders = async (statuses) => {
```

Comment:
Can we move this function to the `withRestApi` flow?

Reply:
Done. Fixed in https://github.com/woocommerce/woocommerce/pull/30184/commits/16083c257478115f613f2e28366a8d16d2f3525f